Backpack/Gradient/README.md

```diff
@@ -0,0 +1,26 @@
+# Backpack/Gradient
+
+## Installation
+
+In `Podfile` add
+
+```
+pod 'Backpack/Gradient'
+```
+
+and then run `pod install`.
+
+## Usage
+
+`Backpack/Gradient` contains the Backpack gradient definitions in the class `BPKGradient`. `BPKGradient` has the two properties:
+
+* `colors` of type `NSArray<UIColor *> *`
```

Comment:
Can we move props to a different section like we do with react/react native components?

Reply:
I think we should just remove these all together in favour of API docs in code